### ISSUE TITLE
Strange frames align on Mac

### ISSUE THREAD

**Issue by widgetii** [elfmz/far2l#302]
I'm using Anonymous Pro font ( https://www.marksimonson.com/fonts/view/anonymous-pro ) in FAR and it seems that frames in several lines are shifted:

<img width="1162" alt="2017-10-06 9 48 09" src="https://user-images.githubusercontent.com/6576495/31271367-f39c2004-aa8f-11e7-9e86-4cc55a56c3b5.png">

$ cat ~/.config/far2l/font
1;18;76;90;90;0;0;Anonymous Pro;56

When I use default PT Mono font (which selected while first launch) everything is ok

**Comment by lieff** [comment]
Yes, this is known problem. Even mono-spaced fonts may not work or work only with limited size sets.

**Comment by widgetii** [comment]
is this problem is wxWidgets?

**Comment by lieff** [comment]
Yes, and performance is another issue. I'm currently working on NVIDIA Path Rendering + NanoVG portable solution for this.

**Comment by widgetii** [comment]
Wow. I will be awesome ))

**Comment by Nomad1** [comment]
Empty file `.config/far2l/nobuffering` fixes the issue.
So you can just ``touch ~/.config/far2l/nobuffering`` to have such picture:
![2018-08-31 14 50 21](https://user-images.githubusercontent.com/3510066/44910809-435b7180-ad2d-11e8-8f3f-d3a759652bf4.png)
